.github/CONTRIBUTING.md

```diff
@@ -99,6 +99,53 @@ To create revisions, you can use [`alembic.sh`](bin/alembic.sh) in the `bin` dir
 
 If you are unfamiliar with alembic, [`here is some information`](/alembic/README). Check also the [documentation](https://alembic.sqlalchemy.org/en/latest/tutorial.html#create-a-migration-script) as well.
 
+## Configuration and environnement
+
+In order to run the bot without any issue, there is some prerequisites.
```

Comment:
Prerequisite is a countable form, therefore you should use "there are some" and not "there is some" (use "is" only if some is followed by an uncountable noun, e.g. "help").